Client/src/App.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import Header from './Header.jsx';
+import Overview from './Overview.jsx';
+
+const ListingHeader = styled.div`
+white-space: nowrap;
+overflow: hidden;
+display: flex;
+flex-wrap: wrap;
+justify-content: space-between;
+margin-top: -20px;
+box-sizing: border-box;
+font-weight: 400;
+-webkit-font-smoothing: antialiased;
+font-family: "Calibre", Helvetica, Arial, sans-serif;
+font-size: 16px;
+line-height: 1.42;
+color: #333333;
+`;
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      campId: 0,
+      campsite: {},
+      host: {},
+      body: '',
+      campers: [],
+    };
+  }
+
+  componentDidMount() {
+    const { pathname } = window.location;
+    fetch(`http://localhost:3005/campsite/${pathname.slice(-3)}`)
```

Comment:
I think this code can be cleaned up a little by using 'window.location.pathname' instead of window.location and slicing out the path of the window location